pom.xml

```diff
@@ -40,6 +44,10 @@
 			<artifactId>postgresql</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
```

Comment:
what does this module do?

Reply:
This one imports the annotations to validate the DTO objects information, with the @Valid tag, tags like @NotNull, @Max, @Min, etc